content/enable-hibernation.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Enable Hibernation (Suspend to Disk)
+description: >
+    How to partition your drive and set up hibernation in Pop!_OS.
+keywords:
+  - Suspend
+  - Suspend to Disk
+  - Hibernate
+  - Hibernation
+  - Swap
+  - Swap Partition
+  - Encrypted Swap
+  - Encryption
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+---
+
+## State of Hibernation on Pop!_OS
+
+Hibernation is not currently enabled on Pop!_OS by default. The factors for this choice are:
```

Comment:
The factor for this choice are >> There are several limitations that keep this from being a default:

Reply:
Fixed.